### PR TITLE
Sync k/features owners with k/sig-release

### PR DESCRIPTION
**Pull request by spiffxp**
Larger pool of people capable of approving PRs to this repo

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
+
 approvers:
 - idvoretskyi
-- philips
-- aronchick
-- apsinha
-- quinton-hoole
-- goltermann
+- spiffxp
+- sig-release-leads
```
Comment (idvoretskyi) [inline OWNERS]: - sig-pm-leads to be added

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# See the OWNERS docs: https://git.k8s.io/community/docs/devel/owners.md
+
+aliases:
```
Comment (idvoretskyi) [inline OWNERS_ALIASES]: - sig-pm-leads to be added:
  - idvoretskyi
  - apsinha
  - calebamiles

Comment (spiffxp) [reply OWNERS_ALIASES]: done

**Comment by spiffxp** [comment]
Should be the last manual merge...

**Pull request merged**